### PR TITLE
GH-92584: Redirect macOS package installation to the PPUG

### PR DESCRIPTION
**Pull request by AA-Turner**
* Issue: gh-92584

----
:books: Documentation preview :books:: https://cpython-previews--108044.org.readthedocs.build/

### CODE REVIEW

File: Doc/using/mac.rst [hunk 1]
```diff
@@ -125,13 +125,11 @@ http://www.hashcollision.org/hkn/python/idle_intro/index.html.
 Installing Additional Python Packages
 =====================================
 
-There are several methods to install additional Python packages:
+.. note::
```
Comment (vstinner) [inline Doc/using/mac.rst]: I don't think that it should be displayed as a note.

Comment (AA-Turner) [inline Doc/using/mac.rst]:
```suggestion
```

File: Doc/using/mac.rst [hunk 2]
```diff
@@ -125,13 +125,11 @@ http://www.hashcollision.org/hkn/python/idle_intro/index.html.
...
 
-* Packages can be installed via the standard Python distutils mode (``python
-  setup.py install``).
+   This section has moved to the `Python Packaging User Guide`_.
```
Comment (AA-Turner) [inline Doc/using/mac.rst]:
```suggestion
This section has moved to the `Python Packaging User Guide`_.
```

File: Doc/using/mac.rst [hunk 3]
```diff
@@ -125,13 +125,11 @@ http://www.hashcollision.org/hkn/python/idle_intro/index.html.
...
 
-* Many packages can also be installed via the :program:`setuptools` extension
-  or :program:`pip` wrapper, see https://pip.pypa.io/.
+   .. _Python Packaging User Guide: https://packaging.python.org/en/latest/tutorials/installing-packages/
```
Comment (AA-Turner) [inline Doc/using/mac.rst]:
```suggestion
.. _Python Packaging User Guide: https://packaging.python.org/en/latest/tutorials/installing-packages/
```

File: Doc/using/mac.rst [hunk 4]
```diff
@@ -125,13 +125,10 @@ http://www.hashcollision.org/hkn/python/idle_intro/index.html.
 Installing Additional Python Packages
 =====================================
 
-There are several methods to install additional Python packages:
```
Comment (AA-Turner) [inline Doc/using/mac.rst]:
```suggestion
```

File: Doc/using/mac.rst (hunk 1 above)
Comment (AA-Turner) [reply Doc/using/mac.rst]: Done (on mobile, terrible interface for applying changes!)

**Comment by vstinner** [comment]
LGTM

**Review by ned-deily — COMMENTED** [review]
LGTM. Thanks, @AA-Turner!

**Pull request merged**